packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -617,6 +617,9 @@ Router = Router.extend({
         }
       );
     },
+    'signout_sync(/)': function () {
```

Comment:
Why do we need a new page/route for this, can we just create the component and [return it instead of Settings](https://github.com/mozilla/fxa/blob/68b599c5923341bf16282932b3f250c18972b53f/packages/fxa-settings/src/components/App/index.tsx#L240)?

Reply:
I thought about just returning a different component, but decided to do a page for the following reasons:

1. We get a view metric.
2. When it comes to do auto login, or support a login button it's more isolated.
3. IMO, if you aren't logged in, you shouldn't be able to end up on the /settings page. 
4. The messaging has nothing to do with the setting page. It just feels like a very separate thing to me.

Reply:
1. We could easily emit a new Glean event in a component though, it doesn't need to be its own page. I think that `usePageViewEvent` function you used is for Amplitude only.
2. I could see this, but I'm also not sure if we want this UX, but I think it would be nice. In that Slack thread I linked in another comment, Mark was asking what cases users can get into this with our fixes. I'm wondering if we won't want to iterate on the approach since it will be so edgecasey.
3. I'd say users inside the browser won't be able to see the URL, but if they do that new tab link out linked in that Slack thread, they will, so point taken.
4. Related to 3, so yeah, point taken.

I'll leave it up to you, was just curious what the rationale was. 🙂 (side note: I don't think putting this behind `/settings/sync_signout` is better since it would be a page in settings not requiring a session token, but dang, it would prevent you from having to do all the content-server route modifications.)

Reply:
I just ended up going with the component approach. This is an edge case, and you are right it touches way less files.